_posts/en/newsletters/2020-02-12-newsletter.md

```diff
@@ -0,0 +1,111 @@
+  part of the workshop also consisted of critical feedback from the
+  audience and replies to the criticism.  A final discussion covered how
+  and when to attempt to get BIP119 activated, including when a PR for
+  it should be opened to the Bitcoin Core repository, what activation
+  mechanism it should use (e.g.  [BIP9][] versionbits), and what range
+  of activation dates would be appropriate if it uses a miner-activated
+  soft fork mechanism such as BIP9.
+
+    Subsequent to
+    the workshop, CTV proposer Jeremy Rubin announced a [mailing
+    list][ctv mailing list] to help coordinate future review and
+    discussion of the BIP119 proposal.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #17578][] FIXME:dongcarl rpc: deprecate getaddressinfo label
+
+- [Bitcoin Core #18032][] extends the results of the `createmultisig`
+  and `addmultisigaddress` RPCs to include a `descriptor` field that
+  contains an [output script descriptor][topic output script descriptors]
+  for the created multisig address.  Descriptors make it easy to copy
+  essential information related to the address that will be necessary to
+  create unsigned transactions that spend money received to the address.
+
+- [C-Lightning #3475][] allows a plugin hook to return `{ "result" :
+  "continue" }` to tell `lightningd` to process the action the same way
+  it would without the hook being executed.  This makes it easy for
+  hooks to only execute in special cases.
+
+- [C-Lightning #3372][] allows the user to specify an alternative
+  program to use instead of one of the default sub-daemons (the
+  C-Lightning system consists of multiple interacting daemons, referred
+  to as *sub-daemons* of `lightningd`).  For example (from the PR
+  description):
+
+      # Use remote_hsmd instead of lightning_hsmd for signing:
+      lightningd --alt-subdaemon=lightning_hsmd:remote_hsmd ...
+
+  This option can be dangerous if the alternative sub-daemon isn't fully
+  compatible with the other daemons being used, but it also allows
+  improved flexibility and may simplify some testing.
+
+- [C-Lightning #3465][] implements anti fee sniping for withdrawal
+  transactions, similar to LND's implementation of the same thing (see
+  [Newsletter #18][news18 afs]).  Anti fee sniping uses the nLockTime
+  field to prevent a transaction from being included in any block whose
+  height is lower than that of the tip of the block chain when the
+  transaction was produced.  This limits the ability of a miner who is
+  reorganizing (forking) the chain from being able to arbitrarily
+  rearrange transactions to maximize their fee revenue.
```

Comment:
If you used that argument with my mother, she might've said, "would you also jump off a bridge because it would make you less distinguishable from a Bitcoin Core user?"  :-)

In seriousness, the reason I didn't mention the distinguishability argument here is because we mentioned it in last week's newsletter:

> A suggestion to set the nLockTime field’s value to a recent or upcoming block height in order to implement anti-fee sniping which helps disincentivize block chain reorganizations and which will help funding transactions blend in with other wallets that have already implement anti-fee sniping (including LND’s sweeping mode, see Newsletter #18).

Additionally, I'm hoping to add anti fee sniping as a topic in our next topic update (which is overdue), so I worry about repeating the same points excessively.